src/pages/ReimbursementAccount/NonUSD/Finish/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import {View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+import Button from '@components/Button';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import * as Expensicons from '@components/Icon/Expensicons';
+import * as Illustrations from '@components/Icon/Illustrations';
+import ScreenWrapper from '@components/ScreenWrapper';
+import ScrollView from '@components/ScrollView';
+import Section from '@components/Section';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@navigation/Navigation';
+import * as Report from '@userActions/Report';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+
+function Finish() {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    const [reimbursementAccount] = useOnyx(ONYXKEYS.REIMBURSEMENT_ACCOUNT);
+    const policyID = reimbursementAccount?.achData?.policyID ?? '-1';
+
+    const handleBackButtonPress = () => {
+        Navigation.goBack();
+    };
+    const handleNavigateToConciergeChat = () => Report.navigateToConciergeChat(true);
+
+    return (
+        <ScreenWrapper
+            testID={Finish.displayName}
+            includeSafeAreaPaddingBottom={false}
+            shouldEnablePickerAvoiding={false}
+            shouldEnableMaxHeight
+        >
+            <HeaderWithBackButton
+                title={translate('finishStep.connect')}
+                onBackButtonPress={handleBackButtonPress}
+            />
+            <ScrollView style={[styles.flex1]}>
+                <Section
+                    title={translate('finishStep.letsFinish')}
+                    icon={Illustrations.ConciergeBubble}
+                    containerStyles={[styles.mb8, styles.mh5]}
+                    titleStyles={[styles.mb3, styles.textHeadline]}
+                >
+                    <Text style={[styles.mb6, styles.mt3, styles.textLabelSupportingEmptyValue]}>{translate('finishStep.thanksFor')}</Text>
+                    <Button
+                        iconStyles={[styles.customMarginButtonWithMenuItem]}
+                        text={translate('finishStep.iHaveA')}
```

Comment:
Do think you this text `I have a question` is little bit small? 
<img width="378" alt="Screenshot 2024-11-20 at 14 58 40" src="https://github.com/user-attachments/assets/a2cf6edb-2ab7-47a2-8749-ec30ad710ad6">

cc @Expensify/design

Reply:
you're right - I'll fix it

Reply:
fixed